enterprise/backend/src/metabase_enterprise/data_editing/params.clj

```diff
@@ -0,0 +1,207 @@
+(ns metabase-enterprise.data-editing.params
```

Comment:
Since we're going to use this in the OSS `/action/v2` routes, I think calling this namespace `metabase.actions.describe` makes more sense - in the future.

We can't move it out of this enterprise module yet because of the direct call to the `data-editing` namespace, but what it's doing there is going to get encapsulated in `apply-mapping` when we clean stuff up.